src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -199,27 +198,32 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
       labelHelp={<NetworkSelectHelperPopover />}
     >
       <div data-test-id="network-attachment-definition-select">
-        {hasPodNetwork && !loaded ? (
-          <Loading />
-        ) : (
-          <SelectTypeahead
-            addOption={(value) =>
-              setCreatedNetworkOptions((prev) => [
-                ...prev.filter((option) => option.value !== value),
-                createNewNetworkOption(value),
-              ])
-            }
-            canCreate
-            dataTestId="select-nad"
-            getCreateAction={getCreateNetworkOption}
-            isFullWidth
-            key={selectedFirstOnLoad ? 'select-nad-with-preselect' : 'select-nad-without-preselect'}
-            options={networkOptions}
-            placeholder={t('Select a NetworkAttachmentDefinition')}
-            selectedValue={networkName}
-            setSelectedValue={handleChange}
-          />
+        {hasPodNetwork && (
+          <FormGroupHelperText validated={validated}>
+            <div className="pf-v6-u-text-color-status-danger">
+              {t(
+                'No NetworkAttachmentDefinitions available. Contact your system administrator for additional support.',
+              )}
+            </div>
+          </FormGroupHelperText>
```

Comment:
We need this part only if there is an actual error. is this true? We already have this in the bottom of the `SelectTypeahead`